### PR TITLE
[cronaswap] add cronaswap strategies

### PR DESCRIPTION
**Pull request by CronaChef**
Fixes # .

Changes proposed in this pull request:
- add cronaswap strategies

This is the most common strategy, it returns the balances of the voters for a balances CRONA token
in CronaSwap project(pools, farms, Liquidity, token).

### CODE REVIEW

**Comment by CronaChef** [comment]
te

**Comment by CronaChef** [comment]
reopened

File: src/strategies/cronaswap/index.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits, parseUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import examplesFile from './examples.json';
+
+export const author = 'cronaswap-dex';
```
Comment (ChaituVR) [inline src/strategies/cronaswap/index.ts:6]:
```suggestion
export const author = 'CronaChef';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Plz revert package-lock.json

**Comment by CronaChef** [comment]
> Plz revert package-lock.json

OK revert it. pls check. thanks

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @CronaChef I see changes in package-lock.json, please remove changes from it

image

**Pull request closed without merging**